internal/resolution/manifest/__snapshots__/maven_test.snap

```diff
@@ -26,7 +170,9 @@
     <maven.compiler.source>1.7</maven.compiler.source>
     <maven.compiler.target>1.7</maven.compiler.target>
     <junit.version>4.13.2</junit.version>
-    <zeppelin.daemon.package.base>../bin</zeppelin.daemon.package.base>
+&#x9;  <zeppelin.daemon.package.base>
+&#x9;    ../bin
+&#x9;  </zeppelin.daemon.package.base>
```

Comment:
There seems to be escaped tabs in this test snapshot?

Reply:
this is expected - this is result from `write()` and then we unescape all the tabs before writing to the actual file in `Write()`